src/main.rs

```diff
@@ -69,28 +69,36 @@ fn main() {
 /// A Shop is a collection of 3 cards.
 struct Shop {
     // TOOD: Add the field to this struct
+    cards: [Card; 3],
 }
 
+use std::cmp;
+
 impl Shop {
     /// Get the price of the most expensive card in the shop
     fn most_expensive(&self) -> u32 {
-        todo!()
+        cmp::max(
+            self.cards[0].price,
+            cmp::max(self.cards[1].price, self.cards[2].price),
+        )
     }
 
     /// Get the total damage of all cards in the shop
     fn total_damage(&self) -> u32 {
-        todo!()
+        self.cards[0].damage + self.cards[1].damage + self.cards[2].damage
```

Comment:
This works, but let's see if we can refactor it to use a `for card in &cards` loop so it's a bit more dynamic :)